### ISSUE TITLE
[Feature Request] Dialog keyboard support and don't show delete confirmation.

### ISSUE THREAD

**Issue by j-a-s-o-n-g** [amir1376/ab-download-manager#478]
1.) This dialog currently doesn't have keyboard navigation. I would like to hit enter to delete and tab to swap between options:
 ![Image](https://github.com/user-attachments/assets/fbd5ac5e-69e3-4cb6-9233-5da2e6c6d488)

2.) I would love a setting in the options to not show this and automatically delete items (and not from disk)

**Comment by amir1376** [comment]
Hi @j-a-s-o-n-g  thank you for your suggestion. I will improve it later
